narratives/ncov_sit-rep_ja_2020-03-05.md

```diff
@@ -39,33 +39,31 @@ abstract: "このレポートは、GISAIDとGenbankが公的に共有してい
 
 ### 参考文献:
 
-* [Wikipedia](https://en.wikipedia.org/wiki/2019%E2%80%9320_Wuhan_coronavirus_outbreak) _2020-01-30_ のコロナウイルス感染症流行の要旨
-* [米CDC](https://www.cdc.gov/coronavirus/index.html) _2020-01-29_ が提供する資料
+* [Wikipedia](https://en.wikipedia.org/wiki/2019%E2%80%9320_Wuhan_coronavirus_outbreak) のコロナウイルス感染症流行の概要 2020-01-30
+* [米CDC](https://www.cdc.gov/coronavirus/index.html) が提供する資料 2020-01-29
 
 <!-- This is right-side text -->
+
 ```auspiceMainDisplayMarkdown
 
-## COVID-19 資料
+## COVID-19 に関する資料
 
-下記に、COVID-19とその症状を引き起こすウイルスであるSARS-CoV-2の詳細を知るために一読する値する資料を用意しました.
-この情報によって、このNarrativeの中で私たちが提示するデータを解釈するのがより容易になります.
+以下に、COVID-19 とその症例を引き起こすウイルスである SARS-CoV-2 を理解するための参考文献を用意しました。この情報により、このレポートで提示するデータの解釈が容易になります。
 
 <div>
   <a href="https://nextstrain.org/help/coronavirus/human-CoV"><img alt="microscopy image of coronaviruses" width="100" src="https://nextstrain.org/static/ncov_narrative-76cfd610d11ef708d213a3170de9519f.png"/> コロナウイルスの背景 </a>
 
-  <a href="https://nextstrain.org/help/coronavirus/SARS-CoV-2"><img alt="illustration of a coronavirus" width="100" src="http://data.nextstrain.org/img_nCoV-CDC.jpg"/> 最近のCOVID-19流行の背景 </a>
+  <a href="https://nextstrain.org/help/coronavirus/SARS-CoV-2"><img alt="illustration of a coronavirus" width="100" src="http://data.nextstrain.org/img_nCoV-CDC.jpg"/> 最近の COVID-19 流行の背景 </a>
 
   <a href="https://nextstrain.org/help/general/how-to-read-a-tree"><img alt="cartoon of a phylogenetic tree" width="100" src="http://data.nextstrain.org/img_toy_alignment_mini.png"/> 系統発生の読み方 </a>
 
 </div>
 
-## Nextstrain Narrative
+## Nextstrain での解析について
 
-このあとのページが[Nextstrain](https://nextstrain.org) を使って行われた解析です
-左側のサイドバーをスクロールしていくと、右側に対応するゲノムデータの可視化とともに段落としてテキストが表示されます.
+このあとのページが[Nextstrain](https://nextstrain.org) を使用して行われた解析です。左側のサイドバーをスクロールすると、項目ごとのテキストが表示され、右側に対応する遺伝子情報が表示されます。
```

Comment:
```suggestion
このあとのページが[Nextstrain](https://nextstrain.org) を使用して行われた解析です。左側のサイドバーをスクロールすると、項目ごとのテキストが表示され、右側に対応するゲノム情報が表示されます。
```

`genome(ゲノム)` has a little bit different meaning from `gene(遺伝子)`.
In this context, it will be valid to use `遺伝子` but we may respect original words.